Sources/Fuzzilli/Compiler/Compiler.swift

```diff
@@ -502,9 +502,16 @@ public class JavaScriptCompiler {
         guard let expr = node.expression else {
             throw CompilerError.invalidASTError("missing concrete expression in expression node")
         }
-
+        
```

Comment:
Nit: I think you introduced some whitespace here and in parser.js, maybe remove that again